insuree/migrations/0020_auto_20240205_0934.py

```diff
@@ -0,0 +1,21 @@
+# Generated by Django 3.2.23 on 2024-02-05 09:34
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('insuree', '0019_auto_20231026_1205'),
+    ]
+
+    operations = [
+        migrations.AddIndex(
+            model_name='family',
+            index=models.Index(fields=['legacy_id', 'validity_from', 'validity_to'], name='tblFamilies_LegacyI_79a19e_idx'),
+        ),
+        migrations.AddIndex(
+            model_name='insuree',
+            index=models.Index(fields=['legacy_id', 'validity_from', 'validity_to', 'chf_id'], name='tblInsuree_LegacyI_6f5da6_idx'),
```

Comment:
Multicolumn indexes should be used sparingly. In most situations, an index on a single column is sufficient and saves space and time. Indexes with more than three columns are unlikely to be helpful unless the usage of the table is extremely stylized.
From PSQL docs: https://www.postgresql.org/docs/current/indexes-multicolumn.html

I know that not every implementation uses psql, but that code will work for everyone so it is worth considering if that many indexes are needed or wanted.